### PR TITLE
Update services version num to 0.36.0-SNAPSHOT

### LINKED ISSUE #4820: Bump services version up to 0.36.0-SNAPSHOT
@tannerjfco reported that integration is currently trying to build HAPI v0.34.0, which is an older version than integration was using (v0.34.1). We forgot to update the services version number when we tagged for version 35.

### PR DESCRIPTION
**Pull request by mhess-swl**
Fixes #4820

### CODE REVIEW

**Review by kimbor — APPROVED** [review]
Looks good!

**Review by Neeharika-Sompalli — APPROVED** [review]
We can merge this once platform version PR is merged

**Review by qnswirlds — APPROVED** [review]

**Pull request merged**
